vnext/Microsoft.ReactNative/Fabric/Composition/CompositionViewComponentView.cpp

```diff
@@ -1245,18 +1245,26 @@ void ComponentView::updateShadowProps(
   // Shadow Properties
   if (oldViewProps.shadowOffset != newViewProps.shadowOffset || oldViewProps.shadowColor != newViewProps.shadowColor ||
       oldViewProps.shadowOpacity != newViewProps.shadowOpacity ||
-      oldViewProps.shadowRadius != newViewProps.shadowRadius) {
+      oldViewProps.shadowRadius != newViewProps.shadowRadius || oldViewProps.boxShadow != newViewProps.boxShadow) {
     applyShadowProps(newViewProps);
   }
 }
 
 void ComponentView::applyShadowProps(const facebook::react::ViewProps &viewProps) noexcept {
   auto shadow = m_compContext.CreateDropShadow();
-  shadow.Offset({viewProps.shadowOffset.width, viewProps.shadowOffset.height, 0});
-  shadow.Opacity(viewProps.shadowOpacity);
-  shadow.BlurRadius(viewProps.shadowRadius);
-  if (viewProps.shadowColor)
-    shadow.Color(theme()->Color(*viewProps.shadowColor));
+  if (!viewProps.boxShadow.empty()) {
+    shadow.Offset({viewProps.boxShadow[0].offsetX, viewProps.boxShadow[0].offsetY, 0});
+    shadow.Opacity(1);
+    shadow.BlurRadius(viewProps.boxShadow[0].blurRadius);
```

Comment:
Heads up that “blur radius” can mean different things depending on implementation.

Gaussian blur follows Gaussian distribution, where standard deviation can be represented as sigma. In blur function, that is number of pixels. The shadow is still visible outside this number of pixels, so different APIs might represent a radius differently.

More concretely:
1. Apple shadow APIs all treat radius as sigma
2. Android and legacy Skia APIs map 1px radius to 0.57px sigma
3. CSS drop shadow filter treats radius as sigma
4. CSS box shadows map 1px radius to 0.5px sigma. https://www.w3.org/TR/css-backgrounds-3/#shadow-blur

What this means in practice… there’s a good chance you need to halve this for box shadows, if composition works in Sigma (or do some special math if they do their own thing to map to css expectations).